### PR TITLE
Add CLI addressTransactions command

### PR DESCRIPTION
**Pull request by BigOokie**
Fixes # NA

Changes:
- add addressTransactions command to CLI. Use /api/v1/transactions

Does this change need to mentioned in CHANGELOG.md?
Yes. Added as part of the PR - review needed

### CODE REVIEW

**Comment by BigOokie** [comment]
Note: This command only works when running against Skycoin Node v0.25.0.
Running on an earlier version of the Node will return a JSON error.

File: src/api/client.go [hunk 1]
```diff
@@ -903,6 +903,20 @@ func (c *Client) PendingTransactionsVerbose() ([]readable.UnconfirmedTransaction
 	return v, nil
 }
 
+// GetTransactions makes a request to POST /api/v1/transactions
+func (c *Client) GetTransactions(addrs []string) ([]readable.TransactionWithStatus, error) {
```
Comment (gz-c) [inline src/api/client.go]: The `Transactions` method already does this

Comment (BigOokie) [reply src/api/client.go]: Will take a look at it now.

Comment (BigOokie) [reply src/api/client.go]: Changes have been pushed. See below.

Comment (BigOokie) [reply src/api/client.go]: Wondering if using `TransactionsVerbose` would be more useful as it provides both `input` and `output` - whereas `Transactions` appears to just provide `output`

Comment (gz-c) [reply src/api/client.go]: The Verbose is probably better to use for this

**Pull request merged**